spec/src/test/java/impl/AnswerResourceTest.java

```diff
@@ -125,8 +141,87 @@ public void shouldThrowInternalErrorWhenCreateAnswerQueryFails() {
             });
     }
 
-    private static Auth newUser() {
-        return new MockAuth(TestSetup.insertUser(userResource).getId());
+    @Test
+    public void getAnswerById() {
```

Comment:
would be great if we can name the test cases with should (something) when (something)